data-reporting-service/investigation-service/src/main/resources/application.yaml

```diff
@@ -3,18 +3,14 @@ spring:
     stream:
       input:
         investigation:
-          topic-name: cdc.nbs_odse.dbo.Investigation
-        defaultData:
-          topic-name: cdc.nbs_odse.dbo.Default.input
+          topic-name: nbs_Public_health_case
       output:
         investigation:
-          topic-name: cdc.nbs_odse.dbo.Investigation.output
-          topic-name-transformed: cdc.nbs_odse.dbo.Investigation.output-transformed
-          topic-name-confirmation: cdc.nbs_odse.dbo.Investigation.Confirmation
-          topic-name-notification: cdc.nbs_odse.dbo.Investigation.Notification
-          topic-name-observation: cdc.nbs_odse.dbo.Investigation.Observation
-    streams:
-      application-id: ${KAFKA_CONSUMER_APP:investigation-reporting-consumer-app}
+          topic-name-reporting: nrt_investigation
+          topic-name-es: elastic_search_investigation
```

Comment:
We can standardize this for Person and Observation as well. I believe they are currently elastic_{}.

Reply:
fixed